README.adoc

```diff
@@ -104,7 +104,7 @@ Your images will also need network connectivity to be able to reach an actual OT
 * `SOTA_DEPLOY_CREDENTIALS` - when set to '1' (default value), deploys credentials to the built image. Override it in `local.conf` to built a generic image that can be provisioned manually after the build.
 * `SOTA_CLIENT_PROV` - which provisioning method to use. Valid options are `aktualizr-shared-prov`, `aktualizr-device-prov`, and `aktualizr-device-prov-hsm`. For more information on these provisioning methods, see the https://docs.ota.here.com/client-config/client-provisioning-methods.html[OTA Connect documentation]. The default is `aktualizr-shared-prov`. This can also be set to an empty string to avoid using a provisioning recipe.
 * `SOTA_CLIENT_FEATURES` - extensions to aktualizr. The only valid options are `hsm` (to build with HSM support) and `secondary-network` (to set up a simulated 'in-vehicle' network with support for a primary node with a DHCP server and a secondary node with a DHCP client).
-* `SOTA_SECONDARY_CONFIG_DIR` - a directory containing JSON configuration files for virtual secondaries on the host. These will be installed into `/etc/sota/ecus` on the device and automatically provided to aktualizr.
+* `SOTA_SECONDARY_CONFIG` - a file containing JSON configuration for secondaries. It will be installed into `/etc/sota/ecus` on the device and automatically provided to aktualizr. See link:https://github.com/advancedtelematic/aktualizr/blob/master/docs/posix-secondaries-bitbaking.adoc[here] for more details.
```

Comment:
Is the bit about /var/sota/ecus accurate? And has aktualizr been bumped already such that the new method is the only way forward now? If yes to both, then this is fine. But how does the config file get on the device? I've forgotten.

Reply:
Yes, I am sure about `/etc/sota/ecus` - this is a folder that the secondaries configuration file is put in.
No, I haven't updated the reference to the desired version of aktualizr at meta-udpater, will do it.

>But how does the config file get on the device?

If `SOTA_SECONDARY_CONFIG` is specified in the local.conf then the corresponding file will be installed into /etc/sota/ecus  folder and /usr/lib/sota/conf.d/30-secondary-config.toml will reference to it.
If `SOTA_SECONDARY_CONFIG` is not specified then it will be auto-generated taking into account other configuration variables, see here for details on them  https://github.com/advancedtelematic/aktualizr/blob/master/docs/posix-secondaries-bitbaking.adoc

Reply:
Bumped the corresponding version of aktualizr.